src/controls/extras/treeitem.coffee

```diff
@@ -34,16 +34,25 @@ class TreeItem #extends Base
   handle_relevant_or_required: ->
     tree_div = @tree.tree_root
     current_node = tree_div.find("[node_value = '" + @value + "']")
-
-    current_node.attr('relevant', @node_relevant())
+    current_node.attr('item-required', @node_required())
```

Comment:
yeah, the schema was updated, but not documentation.  As far as I know the documentation still exists as a pdf file, and really should be updated to a wiki or something.  Since it was just SDPS using the new features and I was working directly with them, I was able to get by not updating it, but now that echo forms will be much more visible, we should take an issue to redo documentation.  Would that be an ECHO issue, or should we start tracking echo forms issues somewhere else?